bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx92-vpn-en.html

```diff
@@ -0,0 +1,188 @@
+            }
+          ) }}
+
+          <p class="refund-policy"><a href="{{ url('products.vpn.landing') }}#faq-refunds">{{ ftl('vpn-shared-refund-policy') }}</a></p>
+          <p class="vpn-pricing-variable-saving">{{ vpn_saving(plan='6-month', lang=LANG) }}</p>
+          <p class="vpn-pricing-variable-total">{{ vpn_total_price(plan='6-month', lang=LANG) }}</p>
+        {% endcall %}
+
+       {% call vpn_content_block(
+         class_name='vpn-pricing-monthly'
+         ) %}
+         <h2 class="vpn-content-block-heading">{{ ftl('vpn-shared-pricing-plan-monthly') }}</h2>
+         <h3 class="vpn-content-block-sub-heading">{{ vpn_monthly_price(plan='monthly', lang=LANG) }}</h3>
+
+         {{ vpn_subscribe_link(
+           entrypoint=_utm_source,
+           link_text=ftl('vpn-shared-pricing-get-monthly'),
+           plan='monthly',
+           class_name='mzp-c-button mzp-t-secondary mzp-t-xl ',
+           lang=LANG,
+           optional_parameters={
+             'utm_campaign': _utm_campaign
+           },
+           optional_attributes={
+             'data-cta-type': 'fxa-vpn',
+             'data-cta-text': 'Get Mozilla VPN monthly',
+           }
+         ) }}
+
+          <p class="refund-policy"><a href="{{ url('products.vpn.landing') }}#faq-refunds">{{ ftl('vpn-shared-refund-policy') }}</a></p>
+        {% endcall %}
+      </div>
+
+      <h2 class="mzp-c-section-heading">A Virtual Private Network from the makers of Firefox</h2>
+
+      <div class="mzp-l-columns mzp-t-columns-three">
+        {% call picto(
+            title='Stay private on public wifi',
+            body=True,
+            image_url='img/firefox/whatsnew/whatsnew92/vpn-private.svg',
+            image_width='330'
+        ) %}
+          <p>Go ahead — check your bank account at the coffee shop. Mozilla VPN provides a secure internet connection for your entire device everywhere you go.</p>
+        {% endcall %}
+
+        {% call picto(
+            title='It’s lightning fast',
+            body=True,
+            image_url='img/firefox/whatsnew/whatsnew92/vpn-speed.svg',
+            image_width='330'
+        ) %}
+          <p>Our WireGuard® powered servers are built for speed so you can stream, download, and game as usual — knowing you’re protected.</p>
+        {% endcall %}
+
+        {% call picto(
+            title='One tap to privacy',
+            body=True,
+            image_url='img/firefox/whatsnew/whatsnew92/vpn-one-tap.svg',
+            image_width='330'
+        ) %}
+          <p>Activate Mozilla VPN with just one tap or click and get an instant, secure connection to one of our 400+ servers located around the world.</p>
+        {% endcall %}
+      </div>
+
+      <p class="vpn-cta">
+        <a class="mzp-c-button mzp-t-product" href="{{ url('products.vpn.landing') }}{{ _params }}" data-cta-text="Learn more about Mozilla VPN" data-cta-type="button">Learn more about Mozilla VPN</a>
```

Comment:
We should avoid using utm params on same-site links as clicking this would then create an all new session in GA. We should use the `?source=whatsnew92` parameter here instead